### ISSUE TITLE
Activation functions

### ISSUE THREAD

**Issue by RameshSrivatsan** [RameshSrivatsan/machine-learning-models#48]
Implement functions such as softmax, relu, tanh functions to provide activations to neural networks from scratch.
